### ISSUE TITLE
ENH: add spaces between genes rather than every 5 mutations

### ISSUE THREAD

**Issue by corneliusroemer** [lenaschimmel/sc2rf#17]
Alternative spacing between genes would be great so one can remember gene boundaries

**Comment by lenaschimmel** [comment]
Yes, sounds like a good option if gene boundaries are important to you. I think to other users, other aspects might be equally or more important, e.g. boundaries of specific amplicons, etc. So I'd like to find a general solution that can be configured to those needs.

Also, I'm not happy with using spacing as the only way for vertical visual guiding. I will experiment with using colored background, but with low-contrast, unintrusive colors. For this to work, I will need 8bit or 24bit color support, and this will need a replacement for the outdated 4bit `termcolor` lib and a lot of testing, and also fallbacks for terminals that only support 4bit colors.
